### PR TITLE
Fix map

### PR DESCRIPTION
**Pull request by TanmayJain92**

### CODE REVIEW

File: src/pages/Shipment/Shipment.js [hunk 1]
```diff
@@ -176,6 +190,24 @@ const Shipment = ({ history }) => {
     }
   }, [selectedShipment, expandedRows]);
 
+  console.log(shipmentData, rows);
```
Comment (patelradhika) [inline src/pages/Shipment/Shipment.js:193]: Remove the console statement.

**Review by patelradhika — CHANGES_REQUESTED** [review]
Once I select a cluster and then click back to global view, I get empty map as shown in the screenshot.
image

File: src/pages/Shipment/Shipment.js (hunk 1 above)
Comment (TanmayJain92) [reply src/pages/Shipment/Shipment.js]: Done

**Comment by TanmayJain92** [comment]
Fixed above issues

**Review by patelradhika — APPROVED** [review]

**Pull request merged**